aspnet/signalr/overview/security/introduction-to-security.md

```diff
@@ -80,6 +80,13 @@ For each request, the server validates the contents of the token to ensure that
 
 Because the connection id is part of the verification process, you should not reveal one user's connection id to other users or store the value on the client, such as in a cookie.
 
+#### Connection Tokens vs other types of tokens
```

Comment:
- Change the heading to "Connection tokens vs. other token types"
- Add a line break after the heading